### DISCUSSION TITLE
error MSB4126: "Installer|MCD" solution configuration is invalid

### DISCUSSION THREAD

**Discussion by gaborschnierer** [Nice3point/RevitTemplates#28] [Q&A] [answered]
Hey guys! 
There was an issue which I managed to solve, thought I'd share, might help others. **Warning**: I'm a newbie, and I am pretty confident that I have no idea I was doing. So all this might make no sense, or be even dangerous, be warned.

Started a new solution from template, added a project from template. Open terminal, type nuke, hit enter. Got this error:
`[filename].sln.metaproj : error MSB4126: The specified solution configuration "Installer|MCD" is invalid. Please specify a valid solution configuration using the Configuration and Platform properties (e.g. MSBuild.exe Solution.sln /p:Configuration=Debug /p:Platform="Any CPU") or leave those properties blank to use the default solution configuration.`

Solution: Using regedit.exe, navigate to the registry path
`HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Session Manager\Environment`
and selected the key Platform. Platform had the value MCD. (in my case)
Then delete only the value MCD. After this the key Platform was still there but empty,  close Regedit, restarted machine and run VS.
[Source](https://helpdrive.wordpress.com/2011/07/10/the-specified-solution-configuration-debugmcd-is-invalid/)

**Comment by Nice3point** [comment]
Weird situation, what OS and IDE do you have?

**Reply by Nice3point** [reply]
the problem is probably related to the "MCD" platform, instead of "Any Platform"

**Reply by Nice3point** [reply, marked as answer]
Usually this key is not in the registry, apparently these are the features of your computer

**Reply by gaborschnierer** [reply]
It is Windows 10 and Visual Studio 2022 Community
